doc/scripting.rst

```diff
@@ -64,8 +65,8 @@ You can interpolate those values with one of supported interpolation methods
 using :func:`interpolate_timeseries` function:
 
 >>> interpolated = interpolate_timeseries(data, method='cubic')
->>> interpolated[7:11].Y
-array([ 151.22663433,  146.80661022,  137.77326894,  127.15995178])
+>>> interpolated[7:11].Y.round(2)
+array([ 151.2271691 ,  146.80723481,  137.77371598,  127.16013142])
```

Comment:
With `np.set_printoptions(precision=1)` above, shouldn't this repr rounded to 1 decimal?